_schema.graphql

```diff
@@ -7969,7 +7975,9 @@ type Sale implements Node {
   href: String
   name: String
   is_auction: Boolean
-  is_benefit: Boolean
+  is_benefit: Boolean @deprecated(reason: "Favor `isBenefit`")
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 7978;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `is_benefit`

Reply:
lol